app/src/pages/inside/stepPage/modals/makeDecisionModal/machineLearningSuggestions/machineLearningSuggestions.jsx

```diff
@@ -160,28 +159,19 @@ export const MachineLearningSuggestions = ({
                 </span>
               </div>
               <div className={cx('suggestion-item-wrapper')} key={testItemResource.id}>
-                <ItemHeader
+                <TestItemDetails
                   item={testItemResource}
                   selectItem={selectMachineLearningSuggestionItem}
                   isSelected={modalState.source.id === testItemResource.id}
                   onClickLinkEvent={onClickExternalLinkEvent}
+                  logs={logs}
+                  suggestRs={suggestRs}
+                  mode={ANALYZED_ITEM}
```

Comment:
You are using `ANALYZED_ITEM` here but it is ML section and looks unclear. In case of using `mode={ANALYZED_ITEM}` in ML section and for real analyzed item please use more general name